app/controllers/api/v1/search_controller.rb

```diff
@@ -677,8 +683,10 @@ def self.match_studies_by_facet(query_results, search_facets)
             facet_name = key.to_s.chomp('_val')
             matching_filter = match_results_by_filter(search_result: result, result_key: key, facets: search_facets)
             matches[accession][facet_name] ||= []
-            matches[accession][facet_name] << matching_filter unless matches[accession][facet_name].include?(matching_filter)
-            matches[accession][:facet_search_weight] += 1
+            if !matches[accession][facet_name].include?(matching_filter)
+              matches[accession][facet_name] << matching_filter
+              matches[accession][:facet_search_weight] += 1
```

Comment:
If I read this correct, we only bump the weight if the facet hadn't already been accounted for.  But if a facet occurs in both the description *and* the metadata, maybe that should that give it extra weight?  I dunno

Reply:
This is only matching by the actual facet filter ID value, meaning the hits are coming from BigQuery.  So we only want to count the filter value once per study.  The inferred matches happen in a different place and count all instances of the value in both name and description.